db/migrate/20170529142557_unify_container_definition_and_container.rb

```diff
@@ -0,0 +1,159 @@
+class UnifyContainerDefinitionAndContainer < ActiveRecord::Migration[5.0]
+  class ContainerDefinition < ActiveRecord::Base
+    has_many :container_port_configs
+    has_many :container_env_vars
+    has_one :security_context, :as => :resource
+    has_one :container
+    self.inheritance_column = :_type_disabled
+  end
+
+  class Container < ActiveRecord::Base
+    belongs_to :container_definition
+    self.inheritance_column = :_type_disabled
+  end
+
+  class SecurityContext < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  class ContainerEnvVar < ActiveRecord::Base
+    belongs_to :container_definition
+    self.inheritance_column = :_type_disabled
```

Comment:
We were asked to put this in all models to prevent problems if we ever add the column in the future (I understand that is the current policy)